example/gardener-local/kind/cluster/values.yaml

```diff
@@ -0,0 +1,17 @@
+environment: skaffold # skaffold|local
+image: kindest/node:v1.24.7
+
+gardener:
+  controlPlane:
+    deployed: true
+  seed:
+    istio:
+      listenAddress: 127.0.0.1
+
+registry:
+  deployed: true
+  hostname: gardener-local-control-plane
+
+#workers:
```

Comment:
Do we need this ?

Reply:
This `values.yaml` is just like an "example" file with defaults to show what is possible to configure, so it's good to specify all potential values here.